tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -794,9 +794,11 @@ ifneq ($(findstring $(EXPLICITLY_SPECIFIED_TEST),$(MICROLITE_TEST_SRCS)),)
   MICROLITE_TEST_SRCS := $(filter-out $(EXPLICITLY_SPECIFIED_TEST), $(MICROLITE_TEST_SRCS))
   EXPLICITLY_SPECIFIED_TEST_SRCS := \
   $(EXPLICITLY_SPECIFIED_TEST) \
-  tensorflow/lite/micro/kernels/conv_test_common.cc
+  tensorflow/lite/micro/kernels/conv_test_common.cc \
```

Comment:
Thoughts about putting all `*_test_data*` under lite/micro/kernels/testdata? I'm wondering if that might help avoid polluting the kernels/ directory once we add more separate test data files.

Reply:
Sounds like a good suggestion to me.  We have a few *_generated_data* that seems to be for unit test as well and also under kernel folder.  If we decide have a lite/micro/kernels/testdata, we should probably move those as well? 

Another point that suggestion is welcome is the use of namespace to separate test data: this CL puts test data into tflite::testing::conv to avoid contaminate of common global test variables.  The way that our test is constructed seems to be built and run one by one with separate binary and there is no real danger of pollution; but I feel explicit different namespace may reduce mental load in code reading.

Reply:
+1 to a common micro/kernels/testdata and to moving the existing code around to match that.

On the namespace: TfLite and TFLM has historically used nested namespaces but for tflm we are moving towards a flat namespace: http://b/170230530

This does mean that we will need some convention for naming the arrays.